phpunit-integration-rest.xml

```diff
@@ -18,9 +15,4 @@
       <directory>tests/bundle/Functional</directory>
     </testsuite>
   </testsuites>
-  <filter>
-    <whitelist>
-      <directory>src</directory>
-    </whitelist>
-  </filter>
```

Comment:
`<filter>` does not exist in PHPUnit 9. I removed it because we are not using code coverage reports and we will need to swap to `<coverage>` for PHPUnit 9 onwards.